test/parser.jl

```diff
@@ -83,3 +83,14 @@ macro test999_str(args...); args; end
 
 # issue #10994
 @test parse("1 + #= \0 =# 2") == :(1 + 2)
+
+# issue #10910
+@test parse(":(using A)") == :(:(using A))
```

Comment:
I'm not sure this really counts as a test of the parser; it only tests that the `parse` function does the same thing as the parser used to read source files. You have to examine the parsed expression in some other way.